README.md

```diff
@@ -101,29 +101,50 @@ For every API following, these inputs are needed in request for authentication:
+| [Err]AddressMismatchedApp       | (地址与app不匹配) the address is not belonging to the App                                                  | make sure that you are using a proper address of your account.                                                                                                        |
+| [Err]FromAndToNotBelonging      | (from和to地址都不是dabank管理的地址) both `from` address and `to` address are not being managed by DABank. | (from和to地址至少得有一个是被dabank管理着的) at least one of the addresses must be managed by DABank.                                                                 |
+| [Err]ServiceInternalError       | server internal error                                                                                      |                                                                                                                                                                       |
+| [Err]BadRequest                 | invalid parameter                                                                                          | (根据详细错误信息检查请求参数) see the response data of `err_info`                                                                                                    |
+| [Err]Unauthorized               | unauthorized                                                                                               | (检查鉴权所需参数是否正确设置) check the credential parameters.                                                                                                       |
+| [Err]UnsupportedSymbol          | (不支持该Symbol) unsupported symbol                                                                        | (停止该币种的支持) you should stop supporting this symbol.                                                                                                            |
+| [Err]BannedSymbol               | (symbol已临时禁用) the symbol is banned temporarily                                                        | (暂时停止该币种的支持) you should stop supporting this symbol temporarily.                                                                                            |
+| [Err]MalformedAddress           | invalid address                                                                                            | (检查请求参数中的地址是否正确。) check the address in parameters according to the address rules of the symbol.                                                        |
+| [Err]ErrIncorrectTransferAmount | invalid amount                                                                                             | (确保交易金额大于0；精度要符合对应币种的规格。) check if the transfer amount is greater than 0, and it is formated with the proper precision according to the symbol. |
```

Comment:
assure the transfer amount is higher than 0, and precision of it  should be consistent with format of the symbol.